reView/utils/bespoke.py

```diff
@@ -87,11 +87,7 @@ def to_wgs(self, rdf):
             self.src_crs,
             always_xy=True
         )
-        lons, lats = transformer.transform(xs, ys)
-        rdf["longitude"] = lons
-        rdf["latitude"] = lats
-        del rdf["x"]
-        del rdf["y"]
+        rdf[['longitude', 'latitude']] = transformer.transform(xs, ys)
```

Comment:
It looks like there were two issues here. Referencing the columns as you did resolves one, and the other was the ValueError that you found. That was because I was counting on the following line `rdf = rdf[self.df.columns]` to remove the 'x' and 'y' columns. But that was a mistake and was causing the "Columns must be same length as key" issue. In the end, I also replaced the two `del rdf['x']` (and 'y') calls with assignment of the values in your way, but to the 'x' and 'y' columns, then renaming those columns (in place) to be "longitude" and "latitude". It's a small difference, but I saw it amounted to about a 1.5% speedup.